spec/index.html

```diff
@@ -867,89 +865,37 @@ <h2>SERVICE Variables (Informative)</h2>
-          <dt><span class="doc-ref" id="rfc3987">[RFC3987]</span></dt>
-          <dd><a href="http://www.ietf.org/rfc/rfc3987.txt">RFC 3987</a>, "Internationalized Resource Identifiers (IRIs)", M. Dürst , M. Suignard</dd>
-          <dt><span class="doc-ref" id="UNICODE">[UNICODE]</span></dt>
-          <dd>The Unicode Standard, Version 4. ISBN 0-321-18578-1, as updated from time to time by the publication of new versions. The latest version of Unicode and additional information on
-          versions of the standard and of the Unicode Character Database is available at <a href=
-          "http://www.unicode.org/unicode/standard/versions/">http://www.unicode.org/unicode/standard/versions/</a>.</dd>
-          <dt><span class="doc-ref" id="XML11">[XML11]</span></dt>
-          <dd>Extensible Markup Language (XML) 1.1, J. Cowan, J. Paoli, E. Maler, C. M. Sperberg-McQueen, F. Yergeau, T. Bray, Editors, W3C Recommendation, 4 February 2004,
-          http://www.w3.org/TR/2004/REC-xml11-20040204/ . <a href="http://www.w3.org/TR/xml11/" title="Latest version of Extensible Markup Language (XML) 1.1">Latest version</a> available at
-          http://www.w3.org/TR/xml11/ .</dd>
-          <dt><span class="doc-ref" id="BCP47">[BCP47]</span></dt>
-          <dd>Best Common Practice 47, P. V. Biron, A. Malhotra, Editors, W3C Recommendation, 28 October 2004, http://www.rfc-editor.org/rfc/bcp/bcp47.txt .</dd>
-        </dl>
-      </section>
-      <section id="sec-non-normative-refs">
-        <h3>Other References</h3>
-        <dl class="bib">
-          <dt><span class="doc-ref" id="RESULTS">[RESULTS]</span></dt>
-          <dd><cite><a href="http://www.w3.org/TR/2013/REC-rdf-sparql-XMLres-20130321">SPARQL Query Results XML Format (Second Edition)</a></cite>, D. Beckett, J. Broekstra, Editors, W3C
-          Recommendation, 21 March 2013, http://www.w3.org/TR/2013/REC-rdf-sparql-XMLres-20130321. <a href="http://www.w3.org/TR/rdf-sparql-XMLres/" title=
-          "Latest version of SPARQL Query Results XML Format (Second Edition)">Latest version</a> available at http://www.w3.org/TR/rdf-sparql-XMLres.</dd>
-          <dt><span class="doc-ref" id="TURTLE">[TURTLE]</span></dt>
-          <dd><cite><a href="http://www.w3.org/TR/2013/CR-turtle-20130219/">Turtle: Terse RDF Triple Language</a></cite>, E Prud'hommeaux, G Carothers, Editors, W3C Candidate Recommendation, 19
-          February 2013, http://www.w3.org/TR/2013/CR-turtle-20130219/. <a href="http://www.w3.org/TR/turtle/" title="Latest version of Turtle">Latest version</a> available at
-          http://www.w3.org/TR/turtle/.</dd>
-        </dl>
-      </section>
+      "norm" data-cite="SPARQL12-QUERY#policy-security">SPARQL Protocol</a> Section 3.1 <a class="norm" data-cite="SPARQL12-QUERY#security-original">SPARQL Query</a> Section 21, and <a data-cite="rfc3986#" class="norm">Uniform Resource
```

Comment:
Nit: do we need to use the section numbers explicitely? The numbers are not very stable.

```suggestion
      "norm" data-cite="SPARQL12-QUERY#policy-security">SPARQL Protocol</a> <a class="norm" data-cite="SPARQL12-QUERY#security-original">SPARQL Query</a>, and <a data-cite="rfc3986#" class="norm">Uniform Resource
```

Reply:
Agreed. Is there a common respec approach to this sort of need? There are other section numbers elsewhere in this document as well.

Reply:
AFAIK there is no way to keep the section numbers in the text via respec.
But maybe we don't need them indeed.

Unless @gkellogg knows a way to keep the section numbers in the next?

Reply:
With or without the section numbers, there's a missing comma above. This is @Tpt's suggestion plus that missing comma.
```suggestion
      "norm" data-cite="SPARQL12-QUERY#policy-security">SPARQL Protocol</a>, <a class="norm" data-cite="SPARQL12-QUERY#security-original">SPARQL Query</a>, and <a data-cite="rfc3986#" class="norm">Uniform Resource
```